third-party/download-content-shell/download-content-shell.js

```diff
@@ -64,13 +65,16 @@ function getPlatform() {
   throw new Error(`Unrecognized platform detected: ${process.platform}`);
 }
 
-function findMostRecentChromiumCommit() {
-  // TODO: this code works only if there is a full chromium checkout present.
-  // const commitMessage = shell('git log --max-count=1 --grep="Cr-Commit-Position"').toString().trim();
-  // const commitPosition = commitMessage.match(/Cr-Commit-Position: refs\/heads\/master@\{#([0-9]+)\}/)[1];
-  // return commitPosition;
-  // TODO: make this dynamic.
-  return '822569';
+async function findMostRecentChromiumCommit() {
+  // Fetch the most recent commit from the GitHub Chromium mirror.
+  const githubUrl = 'https://api.github.com/repos/chromium/chromium/commits/master';
+  // https://docs.github.com/en/rest/overview/resources-in-the-rest-api#user-agent-required
+  const options = {headers: {'User-Agent': 'lighthouse'}};
+  const latestCommit = await utils.fetch(githubUrl, options);
+  const commitMessage = JSON.parse(latestCommit).commit.message;
+  const commitPosition = /Cr-Commit-Position: refs\/heads\/master@\{#([0-9]+)\}/.exec(commitMessage)[1];
+
+  return commitPosition;
```

Comment:
we could do something like `return commitPosition - commitPosition % 1000;` or something if we still want this to only update periodically

Reply:
> we could do something like return commitPosition - commitPosition % 1000; or something if we still want this to only update periodically

This seems unnecessary for now unless anyone really thinks they want it. This is often a few seconds even with a download and unzip

Reply:
> This is often a few seconds ...

Not if you're on bad wifi like I currently am :)

Reply:
> what's the ~time delta for 1000 revisions?

there's about 420 revisions per weekday.

🕶️